src/util/util.js

```diff
@@ -34,6 +34,10 @@ export function isWindowsDriver() {
     return (store.getConfig().driver_interface.interface === "TUSB");
 }
 
+export function hasDriverVersion() {
+    return store.getConfig().driver_interface.version !== undefined
```

Comment:
As mentioned on Discord, `Option::None` may resolve to `null` and not `undefined`